diff -ur ../CLAPACK-WA/BLAS/SRC/Makefile ./BLAS/SRC/Makefile
--- ../CLAPACK-WA/BLAS/SRC/Makefile	2020-12-10 17:22:07.801816700 +0000
+++ ./BLAS/SRC/Makefile	2020-12-10 15:20:50.581816700 +0000
@@ -139,7 +139,8 @@
 	$(ZBLAS2) $(ZBLAS3) $(ALLBLAS)
 
 $(BLASLIB): $(ALLOBJ)
-	$(LD) $(CFLAGS) -o $@ $(ALLOBJ)
+	$(ARCH) $(ARCHFLAGS) $@ $(ALLOBJ)
+	$(RANLIB) $(BLASLIB)
 
 single: $(SBLAS1) $(ALLBLAS) $(SBLAS2) $(SBLAS3)
 	$(ARCH) $(ARCHFLAGS) $(BLASLIB) $(SBLAS1) $(ALLBLAS) \
diff -ur ../CLAPACK-WA/BLAS/WRAP/Makefile ./BLAS/WRAP/Makefile
--- ../CLAPACK-WA/BLAS/WRAP/Makefile	2020-12-10 17:22:07.791816700 +0000
+++ ./BLAS/WRAP/Makefile	2020-12-10 15:20:50.581816700 +0000
@@ -5,10 +5,11 @@
 #  This is the makefile to create a wrapper for the CBLAS.
 #######################################################################
 
-all: libcblaswr.bc
+all: libcblaswr.a
 
-libcblaswr.bc: cblaswr.o
-	$(LD) $(LDFLAGS) -o $@ $?
+libcblaswr.a: cblaswr.o
+	$(ARCH) $(ARCHFLAGS) $@ $?
+	$(RANLIB) $@
 
 libfblaswr.a: fblaswr.o
 	$(ARCH) $(ARCHFLAGS) libfblaswr.a fblaswr.o
diff -ur ../CLAPACK-WA/F2CLIBS/libf2c/Makefile ./F2CLIBS/libf2c/Makefile
--- ../CLAPACK-WA/F2CLIBS/libf2c/Makefile	2020-12-10 17:22:07.791816700 +0000
+++ ./F2CLIBS/libf2c/Makefile	2020-12-10 16:45:33.981816700 +0000
@@ -68,10 +68,12 @@
 OFILES = $(MISC) $(POW) $(CX) $(DCX) $(REAL) $(DBL) $(INT) \
 	$(HALF) $(CMP) $(EFL) $(CHAR) $(I77) $(TIME)
 
-all: f2c.h signal1.h sysdep1.h libf2c.bc clapack_install
+all: f2c.h signal1.h sysdep1.h libf2c.a clapack_install
+
+libf2c.a: $(OFILES)
+	$(ARCH) $(ARCHFLAGS) libf2c.a $?
+	$(RANLIB) libf2c.a
 
-libf2c.bc: $(OFILES)
-	$(LD) $(LDFLAGS) -o libf2c.bc $?
 
 ## Shared-library variant: the following rule works on Linux
 ## systems.  Details are system-dependent.  Under Linux, -fPIC
@@ -115,14 +117,14 @@
 #main.o: main.c
 #	$(CC) -c -Donexit=on_exit -DSkip_f2c_Undefs main.c
 
-install: libf2c.bc
-	cp libf2c.bc $(LIBDIR)
+install: libf2c.a
+	cp libf2c.a $(LIBDIR)
 
-clapack_install: libf2c.bc
-	mv libf2c.bc ..
+clapack_install: libf2c.a
+	mv libf2c.a ..
 
 clean:
-	rm -f libf2c.bc *.o arith.h signal1.h sysdep1.h
+	rm -f libf2c.a *.o arith.h signal1.h sysdep1.h
 
 backspac.o:	fio.h
 close.o:	fio.h
diff -ur ../CLAPACK-WA/Makefile ./Makefile
--- ../CLAPACK-WA/Makefile	2020-12-10 17:22:07.801816700 +0000
+++ ./Makefile	2020-12-10 15:20:50.701816700 +0000
@@ -32,7 +32,7 @@
 	( cd BLAS/SRC; $(MAKE) )
 
 cblaswrap: 
-	( cd BLAS/WRAP; $(MAKE) libcblaswr.bc; cp libcblaswr.bc ../.. )
+	( cd BLAS/WRAP; $(MAKE) libcblaswr.a; cp libcblaswr.a ../.. )
 
 fblaswrap: 
 	( cd BLAS/WRAP; $(MAKE) libfblaswr.a; cp libfblaswr.a ../.. )
@@ -104,4 +104,4 @@
 	( cd TESTING; rm -f xlin* xeig* )
 
 cleanall: cleanlib cleanblas_testing cleantesting 
-	rm -f *.bc TESTING/*.out INSTALL/test*  BLAS/*.out
+	rm -f *.a TESTING/*.out INSTALL/test*  BLAS/*.out
diff -ur ../CLAPACK-WA/SRC/Makefile ./SRC/Makefile
--- ../CLAPACK-WA/SRC/Makefile	2020-12-10 17:22:07.801816700 +0000
+++ ./SRC/Makefile	2020-12-10 16:05:29.631816700 +0000
@@ -379,7 +379,8 @@
 	$(ALLAUX)
 
 ../$(LAPACKLIB): $(ALLOBJ) $(ALLXOBJ)
-	$(CC) $(CFLAGS) -o $@ $(ALLOBJ) $(ALLXOBJ)
+	$(ARCH) $(ARCHFLAGS) $@ $(ALLOBJ) $(ALLXOBJ)
+	$(RANLIB) $@
 
 single: $(SLASRC) $(ALLAUX) $(SCLAUX) 
 	$(ARCH) $(ARCHFLAGS) ../$(LAPACKLIB) $(SLASRC) $(ALLAUX) \
@@ -423,7 +424,7 @@
 	rm -f *.o
 
 .c.o: 
-	$(CC) $(CFLAGS) -c $<
+	$(CC) $(CFLAGS) -c $< -o $@ 
 
 slaruv.o: slaruv.c ; $(CC) $(NOOPT)  -c $< -o $@
 dlaruv.o: dlaruv.c ; $(CC) $(NOOPT)  -c $< -o $@
diff -ur ../CLAPACK-WA/make.inc ./make.inc
--- ../CLAPACK-WA/make.inc	2020-12-10 17:22:07.801816700 +0000
+++ ./make.inc	2020-12-10 17:21:21.671816700 +0000
@@ -26,9 +26,10 @@
 #CC        = gcc  # inherit $CC from emmake
 # if no wrapping of the blas library is needed, uncomment next line
 #CC        = gcc -DNO_BLAS_WRAP
+CFLAGS    = -I$(TOPDIR)/INCLUDE -fPIC -O3
 LOADER    = $(CC)
 LOADOPTS  =
-NOOPT     = -O0 -I$(TOPDIR)/INCLUDE
+NOOPT     = -O0 -I$(TOPDIR)/INCLUDE -fPIC
 #######################################################################
 #
 # Timer for the SECOND and DSECND routines
@@ -48,15 +49,15 @@
 #  The archiver and the flag(s) to use when building archive (library)
 #  If you system has no ranlib, set RANLIB = echo.
 #
-ARCH     = ar
+ARCH     = emar
 ARCHFLAGS= cr
-RANLIB   = ranlib
+RANLIB   = emranlib
 #
 #  The location of BLAS library for linking the testing programs.
 #  The target's machine-specific, optimized BLAS library should be
 #  used whenever possible.
 #
-BLASLIB      = ../../blas$(PLAT).bc
+BLASLIB      = ../../blas$(PLAT).a
 #
 #  Location of the extended-precision BLAS (XBLAS) Fortran library
 #  used for building and testing extended-precision routines.  The
@@ -69,8 +70,8 @@
 #
 #  Names of generated libraries.
 #
-LAPACKLIB    = lapack$(PLAT).bc
-F2CLIB       = ../../F2CLIBS/libf2c.bc
-TMGLIB       = tmglib$(PLAT).bc
-EIGSRCLIB    = eigsrc$(PLAT).bc
-LINSRCLIB    = linsrc$(PLAT).bc
+LAPACKLIB    = lapack$(PLAT).a
+F2CLIB       = ../../F2CLIBS/libf2c.a
+TMGLIB       = tmglib$(PLAT).a
+EIGSRCLIB    = eigsrc$(PLAT).a
+LINSRCLIB    = linsrc$(PLAT).a
